conformance/lang/values-types-and-variables/array.balt

```diff
@@ -0,0 +1,99 @@
+Test-case: output
+Description: Test valid array types and values
+Label: array
```

Comment:
I guess these tests are coveting 5.4.1.1 Array types section. Then Label should be `array-type-descriptor`.  Also, rename the file name.